src/graph.rs

```diff
@@ -796,10 +812,8 @@ impl ModuleSlot {
   }
 }
 
-type ModuleResult<'a> = (
-  &'a ModuleSpecifier,
-  Result<&'a Module, &'a ModuleGraphError>,
-);
+type ModuleResult<'a> =
+  (&'a ModuleSpecifier, Result<&'a Module, &'a ModuleError>);
```

Comment:
It doesn't really make sense to store resolution errors in the module slots, so changed this from `ModuleGraphError` to just `ModuleError`.